commerce_coordinator/apps/lms/views.py

```diff
@@ -95,24 +72,18 @@ def _redirect_response_payment(self, request):
         return redirect
 
     @staticmethod
-    def _add_utm_and_waffle_flags_params_to_url(url, params):
+    def _add_query_params_to_redirect_url(url, params):
         """
-        Add UTM (Urchin Tracking/Google Analytics) flags to the URL for the MFE to use in its reporting,
-        as well as any Waffle Flags
-
+        Add query params to the URL for the MFE to use in its reporting
         Args:
             params (list): Query Params from Req as an encoded list
         Returns:
             url (str): A URL asa Python String
         """
 
-        # utm_params is [(u'utm_content', u'course-v1:IDBx IDB20.1x 1T2017'),... + waffles
-        utm_params = [item for item in params
-                      if QueryParamPrefixes.GOOGLE_ANALYTICS.value or QueryParamPrefixes.WAFFLE_FLAG.value in item[0]]
-        # utm_params is utm_content=course-v1%3AIDBx+IDB20.1x+1T2017&...
-        utm_params = urlencode(utm_params, True)
-        # utm_params is utm_content=course-v1:IDBx+IDB20.1x+1T2017&...
-        # (course-keys do not have url encoding)
-        utm_params = unquote(utm_params)
-        url = url + '?' + utm_params if utm_params else url
+        query_params = list(params)
+        query_params = urlencode(query_params, True)
+        query_params = unquote(query_params)
+        url = url + '?' + query_params if query_params else url
+
         return url
```

Comment:
Would just like to get sign off from @grmartin that this code & related tests are no longer needed - (sorry if I missed an earlier conversation, but I thought we were still going to need Analytics for our release).

Reply:
Had chats with @shafqatfarhan and @aht007 that the purpose of this method is to just extract query params from request url and append to it redirect url. The conversations should fall under the resolved chats in this pr.